bin/git-undo

```diff
@@ -12,26 +34,40 @@ EOL
     read -r res
     case "${res}" in
       "Y" | "y")
-        test $2 -gt 1 > /dev/null 2>&1 && back="~$2"
-        git reset --hard HEAD$back
-        exit $?
+        parm1=hard
+        parm2=${2:-1}
```

Comment:
Better to use more reasonable argument name than `parmX`.

Reply:
I kept them generic on purpose. The names in the function are what matter. If you would like to name the parameters, I'll be glad to use them.